src/role_nextroomer.js

```diff
@@ -308,11 +308,15 @@ roles.nextroomer.settle = function(creep) {
     methods.push(Creep.constructTask);
   }
 
+  // Fill energy after building is done
+  if (creep.room.controller.level >= 4) {
+    methods.push(Creep.transferEnergy);
+  }
```

Comment:
working on a script for transfer energy from hight stored energy room to lowest one. i think it can resolve all energy problems at start.